presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -325,7 +324,6 @@ Aggregations
 .. function:: convex_hull_agg(Geometry) -> Geometry
 
     Returns the minimum convex geometry that encloses all input geometries.
```

Comment:
Documentation change for `convex_hull_agg` needs to be in the same commit as the change itself.